Firefox Selection Fix.sh

```diff
@@ -66,16 +81,43 @@ function fix_firefox(){
   echo
   cd /tmp || exit
   mkdir omni
+
+  ### Cries on macOS some warnings and errors:
+  # warning [/Applications/Firefox.app/Contents/Resources/browser/omni.ja]:
+  #     48078620 extra bytes at beginning or within zipfile (attempting to process anyway)
+  # error [/Applications/Firefox.app/Contents/Resources/browser/omni.ja]:
+  #     reported length of central directory is -48078620 bytes too long (Atari STZip zipfile?  J.H.Holm ZIPSPLIT 1.1 zipfile?).  Compensating...
```

Comment:
What is the purpose of this comment? The fact that these error messages appear is already documented in the README.md.